corehq/apps/users/models.py

```diff
@@ -1287,6 +1287,11 @@ def create(cls, domain, username, password, email=None, uuid='', date='',
         else:
             couch_user.created_on = datetime.utcnow()
 
+        DjangoUserCommCareFields.objects.create(
+            user=django_user,
+            couch_user_id=couch_user._id,
+        )
```

Comment:
just looking at the code above it seems like `django_user` could already exists - that seems quite an edge case but what would happen if 2 of these models got created for one user?